source/_integrations/waze_travel_time.markdown

```diff
@@ -25,6 +25,89 @@ Notes:
 - The string inputs for `Substring *` allow you to force the {% term integration %} to use a particular route or avoid a particular route in its time travel calculation. These inputs are case insensitive and matched against the description of the route.
 - When using the `Avoid Toll Roads?`, `Avoid Subscription Roads?` and `Avoid Ferries?` options, be aware that Waze will sometimes still route you over toll roads or ferries if a valid vignette/subscription is assumed. Default behavior is that Waze will route you over roads having subscription options. It is therefor best is to set both `Avoid Toll Roads?` and `Avoid Subscription Roads?` or `Avoid Ferries?` if needed and experiment to ensure the desired outcome.
 
+## Service `waze_travel_time.get_travel_times`
+
+This service populates [response data](/docs/scripts/service-calls#use-templates-to-handle-response-data)
+with route alternatives and travel times between two locations.
+
+| Service data attribute | Optional | Description | Example |
+| ---------------------- | -------- | ----------- | --------|
+| `origin` | no | The origin of the route | "51.330436, 3.802043" |
+| `destination` | no | he destination of the route | "51.330436, 3.802043" |
+| `region` | no | The region. Controls which waze server is used. | "us" |
+| `units` | yes | Which unit system to use | metric |
+| `vehicle_type` | yes | Which vehicle to use | "car" |
+| `incl_filter` | yes | Which streetnames must be part of the route | "A321" |
+| `excl_filter` | yes | Which streetnames must not be part of the route | "A321" |
+| `realtime` | yes | Use realtime or statistical data | True |
```

Comment:
```suggestion
| `realtime` | yes | Use real-time or statistical data | True |
```
real time as a noun is two words, but [hyphenated when used as an adjective](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/r/real-time).